doc/meetings/2020-04-09.md

```diff
@@ -0,0 +1,68 @@
+# Node.js Release Working Group Meeting 2020-04-09
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=6ZAAIHiaRdc
+* **GitHub Issue**: https://github.com/nodejs/Release/issues/564
+* **Minutes Google Doc**: https://docs.google.com/document/d/1usc9LfhioWNq1d3YmiLlsrdcRmDEXpuZjMFb5Ue7Gpo
+
+## Present
+
+* Beth Griggs (@BethGriggs)
+* Michael Zasso (@targos)
+* Myles Borins (@MylesBorins)
+* Richard Lau (@richardlau)
+* Ruben Bridgewater (@BridgeAR)
+
+## Agenda
+
+## Announcements
+ 
+* All new releases are now notarized on macOS.
+
+### nodejs/Release
+
+* Release cadence due to COVID-19 Pandemic [#553](https://github.com/nodejs/Release/issues/553)
+  * Blog available at https://nodejs.org/en/blog/announcements/adjusted-release-schedule-covid/
+  * Continue to review the schedule.
+
+## Q&A, Other
+
+* Node.js 14 - [#567](https://github.com/nodejs/Release/issues/567)
+  * Due 21st April. 
+  * Build updating toolchains and compilers.
+  * Foundation requesting a blog/marketing.
+* Node.js 13 - [#487](https://github.com/nodejs/Release/issues/487)
+  * 2 more releases.
+  * Last release after Node.js 14 will be patches only.
+* Node.js 12 - [#494](https://github.com/nodejs/Release/issues/494)
+  * Patch release due in April
+  * No volunteers as yet, but will try to land commits on staging.
+  * Make a call if there’s enough on staging to push out a release in April.
+* Node.js 10 - [#504](https://github.com/nodejs/Release/issues/504)
+  * Final release scheduled for early May, if required.
+
+* OpenSSL 1.1.1f  - may require releases of v10.x and v12.x. 
```

Comment:
```suggestion
* OpenSSL 1.1.1g  - may require releases of v10.x and v12.x. 
```

Reply:
I’d keep `f` in the minutes as that’s the one we discussed in the meeting.